doc/PLUGINS.md

```diff
@@ -547,6 +547,44 @@ there's a member `error_message`, that member is sent to the peer
 before disconnection.
 
 
+### `commitment_revocation`
+
```

Comment:
It seems the other hook descriptions always start with `This hook is called when ...`.  For consistency suggest adding the following one-sentence paragraph:

    This hook is called whenever a channel is updated.